src/middlewares/checkUserExist.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable require-jsdoc */
+
+import { User } from "../database/models/index";
```

Comment:
I think we already have a middleware that checks this. If it is there, get rid of this file and use existing middleware.